.ci/aarch64/kubernetes/init.sh

```diff
@@ -20,3 +20,27 @@ ${SCRIPT_PATH}/../../.ci/install_yq.sh
 sudo -E ${GOPATH}/bin/yq d -i -d5 $network_plugin_config_file $memory_resource  > /dev/null
 
 network_plugin_config="$network_plugin_config_file"
+
+wait_pods_ready()
+{
+	# Master components provide the cluster’s control plane, including kube-apisever,
+	# etcd, kube-scheduler, kube-controller-manager, etc.
+	# We need to ensure their readiness on aarch64.
+	local system_pod_wait_time=120
+	local sleep_time=5
```

Comment:
I wonder if `system_pod_wait_time` and `sleep_time` could be sourced from ` integration/kubernetes/init.sh`?

Reply:
Hi~ Since x86_64 also share the same logic to validate master components, not `coredns` pods.
I have made the function general and `system_pod_wait_time` and `sleep_time` not local any more~~~😜